allensdk/test/brain_observatory/ecephys/stimulus_analysis/test_stimulus_analysis.py

```diff
@@ -15,18 +14,23 @@ def ecephys_session():
     units_df = units_df.set_index('unit_id')
     ecephys_ses.units = units_df
     ecephys_ses.spike_times = {uid: np.linspace(0, 1.0, 5) for uid in np.arange(20)}
+    ecephys_ses.running_speed = pd.DataFrame({'start_time': np.linspace(0, 100.0, 20), 'velocity': [0.001]*20})
     return ecephys_ses
 
+
+@pytest.mark.skip()
```

Comment:
so are these todos?

Reply:
A lot of these functions still exists, but are no longer state-free public methods. If/when these classes have been changed back we should be able to still use these tests.